app/presenters/flood_risk_engine/tabular_enrollment_detail/row_builder.rb

```diff
@@ -62,13 +62,13 @@ def location_dredging_length_row
       def organisation_name_row
         build_row name: :organisation_name,
                   value: enrollment_presenter.organisation_name,
-                  step: organisation_type
+                  step: enrollment.org_type
       end
 
       def organisation_address_row
         build_row name: :organisation_address,
                   value: enrollment_presenter.organisation_address,
-                  step: "#{organisation_type}_address"
+                  step: "#{enrollment.org_type}_address"
```

Comment:
`organisation_type` returns a humanised form of the organisation name, but need the symbolised version for links, and that's returned by `enrollment.org_type`